content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,208 @@
+
+Say, we have this alignment stored as a file with the 
+name `alignment.fasta` in our present working directory:
+
+```
+fasta_filename <- "alignment.fasta"
+
+# Check if it really exists
+stopifnot(file.exists(fasta_filename))
+```
+
+To obtain a BEAST2 posterior with `babette`, do:
+
+```
+output <- bbt_run(fasta_filename)
+```
+
+And plotting the posterior trees:
+
+```
+plot_densitree(
+  output$alignment_trees,
+  alpha = 0.01,
+  consensus = as.character(c(1:4)),
+  cex = 2.0,
+  scaleX = TRUE,
+  scale.bar = FALSE
+)
+```
+
+Note that for plotting the posterior trees, I've added many arguments to 
+make it a beautiful blog post picture :-)
+
+#### More about babette
+
+`babette` actually is a suite of five packages, 
+calling and/or offering an alternative for the BEAST2 tool suite:
+
+What|BEAST2 suite|babette suite
+---|---|---
+Create a BEAST2 input file|BEAUti|`beautier`
+Run (only) BEAST2|`beast`|`beastier`
+Visualize posterior parameter estimates|Tracer|`tracerer`
+Visualize posterior phylogenies|DensiTree|`babette`
+Install BEAST2 packages|BEAUti|`mauricer`
+
+The BEAST2 tool suite has - at the moment - more functionality than `babette`.
+The [`babette` GitHub page](https://github.com/ropensci/babette) has a place where feature requests can be and
+are made. If you want something added and you volunteer to help (that is,
+answer my naive questions and testing that feature), let me know!
+
+Although `babette` may not have all BEAST2 functionality,
+`babette` already helped me create thousands of posteriors from an R script!
+Also the analysis done in this blog post, can be found
+[at the babette GitHub](https://github.com/ropensci/babette/blob/master/scripts/ropensci_create_figs_blog_post.R).
+
+#### Why rOpenSci
+
+I enjoy having my academic manuscripts reviewed.
+We expect the articles we read to be clear and the opinion
+of our peers help achieve this for our own work.
+But when I read/run the code in the unreviewed 
+supplementary materials section, there is a loss in
+clarity. Same goes for R packages written by academics:
+the article that describes these packages are well-written,
+but the actual code can be cumbersome to work with.
+To set a good example, I wanted my code reviewed.
+
+I enjoy having my code reviewed. Even when trying to
+write exemplary and spotless code, reviewers usually 
+expose the weaker spots in the software.
+My rOpenSci reviewers did an excellent job at that indeed! 
+Thanks to them, I think `babette` went from a good package to
+an excellent one. 
+
+### Acknowledgements
+
+Yacine Ben Chehida and Paul van Els supplied the first use cases.
+Reviews were provided by David Winter (@dwinter) 
+and Joëlle Barido-Sottani (@bjoelle), 
+with a contribution from Guangchuang Yu (@GuangchuangYu).
+Noam Ross (@noamross) handled the review process. 
+Raphael Scherrer and Jana Riederer helped by submitting bug reports. 
+Not only was code reviewed, this (non-code) blog post was reviewed 
```

Comment:
Thank you for the acknowledgement. It is not necessary to do this for me or for the editor handling your review though. In this case "it's the thought that counts".

Reply:
OK, I will remove the line. Then here: thanks you for your help :+1:

Reply:
... also Noah's :rainbow: